src/views/CharacterBuild.js

```diff
@@ -62,13 +72,7 @@ export default function CharacterBuild() {
                 }}
                 size="small"
                 className={classes.field}
-                onChange={e => {
-                  if (stats === "character_name") {
-                    charBuild[stats] = e.target.value;
-                  } else {
-                    charBuild[stats] = parseFloat(e.target.value);
-                  }
-                }}
```

Comment:
good job cleaning it up, but we still need defaultValue to see the stats

Reply:
what do you mean by needing it to see the stats